Framework/PythonInterface/test/python/plugins/algorithms/WorkflowAlgorithms/PaalmanPingsMonteCarloAbsorptionTest.py

```diff
@@ -22,12 +22,14 @@ def setUpClass(cls):
         Load('irs26173_graphite002_red.nxs', OutputWorkspace='container_ws')
         Load('osi104367_elf.nxs', OutputWorkspace='indirect_elastic_ws')
         Load('ILL_IN16B_FWS_Reduced.nxs', OutputWorkspace='indirect_fws_ws')
+        Load('HRP38692a', OutputWorkspace='elastic_ws')
```

Comment:
Would it be possible to add a test on some direct data here as well. As there seems to be multiple places in the code where Direct has been added later as a seperate case and it would be nice to have some surity it works.